src/iconv.cpp

```diff
@@ -110,7 +110,7 @@ IConv::convert(const std::string& text)
 
     // We try to avoid to much copying around, so we write directly into
     // a std::string
-    tinygettext_ICONV_CONST char* inbuf = const_cast<char*>(&text[0]);
+    const char* inbuf = const_cast<char*>(&text[0]);
```

Comment:
See https://sourceware.org/bugzilla/show_bug.cgi?id=2962 (featuring Ulrich Drepper) and [1](http://stackoverflow.com/questions/11421439/how-can-i-portably-call-a-c-function-that-takes-a-char-on-some-platforms-and) [2](http://pubs.opengroup.org/onlinepubs/9699919799/functions/iconv.html) [3](https://www.gnu.org/savannah-checkouts/gnu/libiconv/documentation/libiconv-1.14/iconv.3.html). The short answer is that some system use const, some don't, they are unlikely to standardize, and using the the wrong version results in a compilation error.